### PR TITLE
Branch-UI-related-features

### PR DESCRIPTION
**Pull request by jltha**
Satisfies #24

### CODE REVIEW

**Review by laiisaac — APPROVED** [review]

File: src/main/java/seedu/sherpass/utills/Parser.java [hunk 1]
```diff
@@ -114,7 +122,7 @@ private static Command prepareMarkOrUnmark(String[] parsedInput, String commandW
 
     private static void printMissingInputMessage(String input, String taskType) {
         System.out.println("Oops! The " + input + " of a(n) " + taskType + " cannot be empty."
-                + HELP_MESSAGE_SPECIFIC_COMMAND);
+            + HELP_MESSAGE_SPECIFIC_COMMAND);
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Parser.java:125]: I think the change needs to be reverted. For code quality.

File: src/main/java/seedu/sherpass/utills/Parser.java [hunk 2]
```diff
@@ -140,14 +148,14 @@ private static void printMissingCommandMessage(String taskType) {
     private static String confirmInvalidDateFormat() throws InvalidInputException {
         Ui anotherUi = new Ui();
         anotherUi.showToUser("It seems that the date and time\nyou gave is not in the correct format.\n"
-                + "Would you like to re-enter a valid date and time? (Y/N)\n"
-                + "*Enter 'No' to skip the adding of this task*");
+            + "Would you like to re-enter a valid date and time? (Y/N)\n"
+            + "*Enter 'No' to skip the adding of this task*");
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Parser.java:152]: Can revert here too.

File: src/main/java/seedu/sherpass/utills/Parser.java [hunk 3]
```diff
@@ -140,14 +148,14 @@ private static void printMissingCommandMessage(String taskType) {
...
         anotherUi.showLine();
         while (true) {
             String input = anotherUi.readCommand();
             anotherUi.showLine();
             if (input.trim().equalsIgnoreCase("Y")
-                    || input.trim().equalsIgnoreCase("Yes")) {
+                || input.trim().equalsIgnoreCase("Yes")) {
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Parser.java:158]: Here too.

File: src/main/java/seedu/sherpass/utills/Parser.java [hunk 4]
```diff
@@ -156,7 +164,7 @@ private static String confirmInvalidDateFormat() throws InvalidInputException {
                 return prepareTaskDate(input.trim(), false);
             }
             if (input.trim().equalsIgnoreCase("N")
-                    || input.trim().equalsIgnoreCase("No")) {
+                || input.trim().equalsIgnoreCase("No")) {
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Parser.java:167]: I think for the rest of this class can revert all the similar changes.

File: src/main/java/seedu/sherpass/utills/Parser.java [hunk 5]
```diff
@@ -305,4 +315,58 @@ public static Command parseCommand(String userInput, TaskList taskList) {
...
+        switch (defaultTimerChoice) {
+        case "1":
+            return DEFAULT_TIMER_ONE;
+        case "2":
+            return DEFAULT_TIMER_TWO;
+        case "3":
+            return DEFAULT_TIMER_THREE;
+        default:
+            ui.showToUser("Sorry! I can't recognise the choice you've entered.\n"
+                + "Please re-enter a valid default timer input");
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Parser.java:328]: Do add 8 spacings here. Can do the same for the rest of the file too.

File: src/main/java/seedu/sherpass/utills/Ui.java [hunk 6]
```diff
@@ -1,17 +1,23 @@
...
-            + "______________________________";
+        + "______________________________";
     private static final String LS = System.lineSeparator();
-    private static final String LOGO = "";
+    private static final String LOGO = "  ____  _\n"
+        + " / ___|| |__   ___ _ __ _ __   __ _ ___ ___\n"
+        + " \\___ \\| '_ \\ / _ \\ '__| '_ \\ / _` / __/ __|\n"
+        + "  ___) | | | |  __/ |  | |_) | (_| \\__ \\__ \\\n"
+        + " |____/|_| |_|\\___|_|  | .__/ \\__,_|___/___/\n"
+        + "                       |_|";
```
Comment (BradenTeo) [inline src/main/java/seedu/sherpass/utills/Ui.java:20]: Can maybe use a different colour here:
Maybe like:
```java
    public static final String ANSI_PURPLE = "\u001B[35m";
    public static final String ANSI_RESET = "\u001B[0m";
    public static final String ALEXIS_ICON =
            ANSI_PURPLE
            + "  ____  _\n"
            + " / ___|| |__   ___ _ __ _ __   __ _ ___ ___\n"
            + " \\___ \\| '_ \\ / _ \\ '__| '_ \\ / _` / __/ __|\n"
            + "  ___) | | | |  __/ |  | |_) | (_| \\__ \\__ \\\n"
            + " |____/|_| |_|\\___|_|  | .__/ \\__,_|___/___/\n"
            + "                       |_|" 
            + ANSI_RESET;
```
This website is useful: [ASC11 art](https://patorjk.com/software/taag/#p=testall&f=Graffiti&t=Sherpass)

**Review by BradenTeo — CHANGES_REQUESTED** [review]
Requested some code quality changes

File: src/main/java/seedu/sherpass/utills/Parser.java (hunk 5 above)
Comment (jltha) [reply src/main/java/seedu/sherpass/utills/Parser.java:328]: For which line do u mean? For the default branch, adding 8 spaces will throw a checkstyle error.

File: src/main/java/seedu/sherpass/utills/Parser.java (hunk 1 above)
Comment (jltha) [reply src/main/java/seedu/sherpass/utills/Parser.java:125]: This was part of the import code. What do u mean by revert?

File: src/main/java/seedu/sherpass/utills/Parser.java (hunk 5 above)
Comment (BradenTeo) [reply src/main/java/seedu/sherpass/utills/Parser.java:328]: Should we ask prof about this? From what I remember, the correct way was to leave 8 spaces.

File: src/main/java/seedu/sherpass/utills/Parser.java (hunk 1 above)
Comment (BradenTeo) [reply src/main/java/seedu/sherpass/utills/Parser.java:125]: Was referring to the 8 spaces

File: src/main/java/seedu/sherpass/utills/Ui.java (hunk 6 above)
Comment (jltha) [reply src/main/java/seedu/sherpass/utills/Ui.java:20]: woah cool cool, I'll try the diff colours

**Review by thedarie — APPROVED** [review]

File: src/main/java/seedu/sherpass/utills/Parser.java (hunk 5 above)
Comment (BradenTeo) [reply src/main/java/seedu/sherpass/utills/Parser.java:328]: I'll approve the PR first and we can edit the code quality afterward.

**Review by BradenTeo — APPROVED** [review]
We'll edit the code quality afterward once prof gets back to us.

**Pull request merged**